lib/Experian/IDAuth.pm

```diff
@@ -334,15 +342,10 @@ sub _get_result_proveid {
     }
 
     # check if client is age verified
```

Comment:
The new version does not seem to mention age verified, how are we checking that now?

Reply:
Sorry, didn't polish off this comment. We have a company-specific rule for checking `age_verified` (which is when kyc_summary_score > 2), so I moved that check to bom-platform instead. There is also a refactoring card where I intend to move `fully_authenticated` to bom-platform as well as I don't believe fields and logic pertaining to those fields belong in a public module.